release-calendar/src/client/models/release-channel.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Release} from '../../types';
+
+export function getMostRecentChannelRTVs(
```

Comment:
Why not rename to `getCurrentReleases`? Someone unfamiliar with the release process wouldn't know what `RTV` is, since the release schema has `name` and not `RTV`. We should have consistent naming.